src/DaPigGuy/libPiggyEconomy/providers/BedrockEconomyProvider.php

```diff
@@ -4,18 +4,18 @@
 
 namespace DaPigGuy\libPiggyEconomy\providers;
 
-use cooldogedev\BedrockEconomy\api\legacy\ClosureContext;
+use cooldogedev\BedrockEconomy\api\type\AsyncAPI;
 use cooldogedev\BedrockEconomy\BedrockEconomy;
 use cooldogedev\BedrockEconomy\api\BedrockEconomyAPI;
-use cooldogedev\BedrockEconomy\api\version\LegacyBEAPI;
-use cooldogedev\BedrockEconomy\currency\CurrencyManager;
+use cooldogedev\BedrockEconomy\currency\Currency;
+use cooldogedev\BedrockEconomy\libs\SOFe\AwaitGenerator\Await;
```

Comment:
Please do not import the shaded virion from BedrockEconomy.

This will cause issues if await-generator is not bundled with BedrockEconomy in cases such as source installations. Additionally, this means no control of the version of await-generator for libPiggyEconomy, which may result in potentially unexpected BC-breaks in the future.